CodeStarterCamp_Week1/Pepero.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Pepero.swift
+//  CodeStarterCamp_Week1
+//
+//  Created by 황지웅 on 2022/04/15.
+//
+import Foundation
+
+func makePeperoBar(peperoHeight: Int) {
+    let rangeOfPeperoHeight = 0..<peperoHeight
+
+    for _ in rangeOfPeperoHeight {
+        print(" | |")
+    }
+}
+
+func makePeperoBody(peperoBody: String, peperoToping: String, peperoBodyHeight: Int) {
+    let rangeOfPeperoHeight = 0..<peperoBodyHeight
+    for height in rangeOfPeperoHeight {
+        makePeperoBodyLayer(peperoBody: peperoBody, peperoToping: peperoToping, peperoHeight: height)
+    }
+}
+
+func makePeperoBodyLayer(peperoBody: String, peperoToping: String, peperoHeight: Int) {
+    if peperoHeight % 2 == 0 {
+        print("\(peperoToping + peperoBody)")
+    }else {
```

Comment:
```suggestion
    } else {
```
else앞뒤로 공백을 넣어주는 코딩 컨벤션이 있습니다!

Reply:
✅